### PR TITLE
Update the pricing page and copy across the website

### PR DESCRIPTION
**Pull request by zchase**
Fixes: https://github.com/pulumi/pulumi-service/issues/6183

This PR updates the pricing page as described in the above issue and updates the copy for our pricing across the website.

### CODE REVIEW

File: themes/default/content/blog/announcing-per-user-pricing-and-unlimited-stacks-for-teams/index.md [hunk 1]
```diff
@@ -3,10 +3,16 @@ title: "Announcing Per User Pricing and Unlimited Stacks for Teams"
...
 authors: ["joe-duffy"]
-tags: 
+tags:
   - features
 ---
 
+{{% notes %}}
+We have updated our pricing structure in response to user feedback and the information
+in this blog post is no longer accurate. Please visit our [Pricing Page]({{< relref "/pricing" >}})
+for the most update to date information about pricing.
```
Comment (cnunciato) [inline themes/default/content/blog/announcing-per-user-pricing-and-unlimited-stacks-for-teams/index.md:13]:
```suggestion
for the most up-to-date information about pricing.
```

File: themes/default/assets/sass/_shape-backgrounds.scss [hunk 2]
```diff
@@ -394,4 +394,46 @@
...
     }
+
+    &.pricing-small-dots {
+        .dot-background-container {
+            @apply w-24 h-full relative;
+
+            .dot-background-overlay {
+                @apply w-full h-full absolute top-0 left-0;
+                background-color: rgba(255,255,255,0.5);
+                z-index: 0;
```
Comment (cnunciato) [inline themes/default/assets/sass/_shape-backgrounds.scss:405]:
```suggestion
                @apply w-full h-full absolute top-0 left-0 z-0;
                background-color: rgba(255,255,255,0.5);
```

File: themes/default/content/blog/pulumi-service-improvements_02-2020/index.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ which now supports a `--version` flag to export older checkpoint files too.
 
 ## Reverse Stack Permissions View
 
-The Team Pro and Enterprise editions support [role-based access control]({{< relref "/docs/intro/console/stack-permissions" >}})
+The Enterprise editions support [role-based access control]({{< relref "/docs/intro/console/stack-permissions" >}})
```
Comment (cnunciato) [inline themes/default/content/blog/pulumi-service-improvements_02-2020/index.md:68]:
```suggestion
The Enterprise Edition supports [role-based access control]({{< relref "/docs/intro/console/stack-permissions" >}})
```

File: themes/default/content/docs/intro/console/_index.md [hunk 4]
```diff
@@ -20,23 +20,20 @@ In your browser, navigate to <a href="https://app.pulumi.com" target="_blank">ap
 <a class="btn btn-secondary" href="https://app.pulumi.com/signup" target="_blank">Sign up</a>
 
 When you sign into the Pulumi Console, an personal account is automatically
-created on the **Community** plan. The Pulumi Community plan is free forever for unlimited individual use.
+created on the **Indvidual** Edition. The Pulumi Individual Edition is free forever for unlimited individual use.
```
Comment (cnunciato) [inline themes/default/content/docs/intro/console/_index.md:23]:
```suggestion
created on the **Individual** Edition. The Pulumi Individual Edition is free forever for unlimited individual use.
```

File: themes/default/content/blog/pulumi-1-0/index.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ Using Pulumi in conjunction with the Pulumi service delivers the following addit
 * **REST API and Webhooks**: a fully programmable web API for advanced automation scenarios.
 * **CI/CD Integrations**: automated deployment integrations with Azure DevOps, AWS Code Services, CircleCI, Codefresh, GitHub, GitLab, Google Cloud Build, Jenkins, Travis, and others.
 
-The Pulumi service is available as a free Community Edition as well as advanced Team and Enterprise SaaS editions for larger teams looking for robust enterprise capabilities.
+The Pulumi service is available as a free Individual Edition as well as advanced Team and Enterprise SaaS editions for larger teams looking for robust enterprise capabilities.
```
Comment (cnunciato) [inline themes/default/content/blog/pulumi-1-0/index.md:43]:
```suggestion
The Pulumi Service is available as a free Individual Edition as well as advanced Team and Enterprise SaaS editions for larger teams looking for robust enterprise capabilities.
```

File: themes/default/layouts/page/pricing.html [hunk 6]
```diff
@@ -200,801 +180,305 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
...
-                    <span>Up to 20</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>Unlimited</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>Unlimited</span>
+                <div class="accordion-item-body-no-animation text-base">
+                    <p>
+                        A Pulumi Credit represents managing 1 resource for 1 hour. 1 resource represents a
```
Comment (cnunciato) [inline themes/default/layouts/page/pricing.html:222]: I get the urge to want to use numbers in these kinds of situations, but it's grammatically incorrect -- in prose, anything under 10 should be spelled out. (I'll only comment on this one instance.)
https://www.grammarly.com/blog/when-to-spell-out-numbers/

**Review by susanev — APPROVED** [review]
i reviewed the marketing pages, the console docs, and the pricing page including the plans, faqs and table and everything looks great except for 1 tiny thing. totally cool to come back later to fix this if needed, not blocking.

- [ ] CI/CD integrations in table should say `Customized for you` under enterprise, instead of `Custom`

File: themes/default/layouts/page/pricing.html [hunk 7]
```diff
@@ -200,801 +180,305 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
...
+                         First, the Pulumi Service is free to use, now and forever, for individuals.
+                         You get all of the convenience of automatic state management, unlimited
+                         deployments, and many other great features without needing to pay anything
+                         at all for it.
+                    </p>
+                    <p>
+                        Second, Pulumi is an <a class="underline" href="https://github.com/pulumi/pulumi">open source project</a>. You can
+                        <a class="underline" href="{{ relref . "/docs/intro/concepts/state#self-managed-backend" }}">run Pulumi entirely
+                        offline</a> without the online service’s features, and manage state yourself, instead of using the online service.
+                        There are no restrictions &mdash; it's all there in the open source for you to use freely as you'd like.
```
Comment (cnunciato) [inline themes/default/layouts/page/pricing.html:337]: "in the open source" feels like a mistake. Maybe this should just be "in the open"?

```suggestion
                        There are no restrictions &mdash; it's all there in the open for you to use freely as you'd like.
```

File: themes/default/layouts/page/pricing.html [hunk 8]
```diff
@@ -200,801 +180,305 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
...
                     <p>
-                        I discovered that Pulumi's community is over 10,000 people and growing, and their customer base
-                        includes a diverse array of companies, from startups to some of the largest Fortune 500 and
-                        Global 2000 organizations. The top two reasons people are choosing Pulumi are 1) introducing a
-                        consistent workflow for delivering and securing infrastructure to any cloud, and 2) empowering
-                        developers and operators to work better together.
+                        I discovered that Pulumi's community is over 10,000 people and growing, and their customer
+                        base includes a diverse array of companies, from startups to some of the largest Fortune 500
+                        and Global 2000 organizations. The top four reasons people are choosing Pulumi are 1) it tames
+                        cloud complexity and reduces infrastructure risks, 2) lets teams use software engineering best
```
Comment (cnunciato) [inline themes/default/layouts/page/pricing.html:444]:
```suggestion
                        cloud complexity and reduces infrastructure risks, 2) it lets teams use software engineering best
```

File: themes/default/layouts/page/pricing.html [hunk 9]
```diff
@@ -200,801 +180,305 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
...
+                        practices with infrastructure, 3) helps teams adopt modern cloud architectures, and 4) increases
```
Comment (cnunciato) [inline themes/default/layouts/page/pricing.html:445]:
```suggestion
                        practices with infrastructure, 3) it helps teams adopt modern cloud architectures, and 4) it increases
```

File: themes/default/layouts/page/pricing.html [hunk 10]
```diff
@@ -1012,124 +496,679 @@ <h5>Sample Email</h5>
...
-                        and manage state yourself, instead of using the online service. There are no restrictions &mdash;
-                        it's all there in the open source for you to use freely as you'd like.
-                    </p>
-                    <p>
-                        Finally, all paid editions offer a 30-day free trial. If this isn't a sufficient duration for
-                        your project, <a class="link" href="{{ relref . "/contact.md" }}">please contact us</a>, and we are happy to work
-                        out a custom trial period for you that makes sense given your proof-of-concept timeline. After your
-                        trial expires, no data will be lost, and there is a grace period with soft enforcement.
+                        You can keep track of current credit usage and upcoming charges by navigating to
+                        Settings and then Billing & Usage in the Pulumi console.
```
Comment (cnunciato) [inline themes/default/layouts/page/pricing.html:505]:
```suggestion
                        Settings and then Billing & Usage in the Pulumi Console.
```

File: themes/default/layouts/page/pricing.html [hunk 11]
```diff
@@ -1012,124 +496,679 @@ <h5>Sample Email</h5>
...
-                    </p>
-                    <p>
-                        All editions are available as annual subscriptions. You may purchase monthly
-                        subscriptions for an additional 20% per month.
-                    </p>
-
-                    <p>
-                        For large numbers of users and custom pricing, support options, on-premises, or to
-                        discuss training and services options, please <a class="link" href="#contact">contact us</a>.
+                        The <a class="underline" href="{{ relref . "open-source-free-tier/" }}">Open-Source Free Tier</a> allows
```
Comment (cnunciato) [inline themes/default/layouts/page/pricing.html:525]:
```suggestion
                        The <a class="underline" href="{{ relref . "open-source-free-tier/" }}">Open-Source Free Edition</a> allows
```

**Review by cnunciato — COMMENTED** [review]
Buncha little-ish things we'll probably want to address, and the Get Started button below the grid looks a bit borked on mobile -- but otherwise, 👍! Thanks a ton for doing this.

image

File: themes/default/layouts/page/pricing.html (hunk 6 above)
Comment (susanev) [reply themes/default/layouts/page/pricing.html:222]: @gthuang @joeduffy thoughts on this?

File: themes/default/layouts/page/pricing.html (hunk 11 above)
Comment (susanev) [reply themes/default/layouts/page/pricing.html:525]: i do no think we should change this, it is the heading on the page: https://www.pulumi.com/pricing/open-source-free-tier/

File: themes/default/layouts/page/pricing.html (hunk 6 above)
Comment (joeduffy) [reply themes/default/layouts/page/pricing.html:222]: Agree with the feedback.

**Review by susanev — APPROVED** [review]

File: themes/default/layouts/page/pricing.html [hunk 12]
```diff
@@ -200,801 +180,305 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
...
+                        <a class="underline" href="{{ relref . "/docs/intro/concepts/resources#custom-resources" }}">custom resource</a>
+                        (e.g., an Amazon S3 bucket), a <a class="underline" href="{{ relref . "/docs/intro/concepts/resources#components" }}">component resource</a>
+                        which is a logical grouping of resources (e.g., an Amazon EKS cluster), or a
+                        <a class="underline" href="{{ relref . "/docs/intro/concepts/stack" }}">stack</a> that contains resources
+                        (e.g., dev, test, prod stacks). You use 1 Pulumi Credit to manage each resource for an hour.
+                    </p>
+                    <p>
+                        For example, the cost of managing 1 stack containing 1 S3 bucket and 1 EC2 instance is
+                        3 Pulumi Credits per hour.
+                    </p>
```
Comment (susanev) [inline themes/default/layouts/page/pricing.html:232]:
```suggestion
                        A Pulumi Credit represents managing one resource for one hour. One resource represents a
                        <a class="underline" href="{{ relref . "/docs/intro/concepts/resources#custom-resources" }}">custom resource</a>
                        (e.g., an Amazon S3 bucket), a <a class="underline" href="{{ relref . "/docs/intro/concepts/resources#components" }}">component resource</a>
                        which is a logical grouping of resources (e.g., an Amazon EKS cluster), or a
                        <a class="underline" href="{{ relref . "/docs/intro/concepts/stack" }}">stack</a> that contains resources
                        (e.g., dev, test, prod stacks). You use one Pulumi Credit to manage each resource for an hour.
                    </p>
                    <p>
                        For example, the cost of managing one stack containing one S3 bucket and one EC2 instance is
                        three Pulumi Credits per hour.
                    </p>
```

this is follow-up to @cnunciato's comment

File: themes/default/layouts/page/pricing.html [hunk 13]
```diff
@@ -200,801 +180,305 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
...
+                <div class="accordion-item-body-no-animation text-base">
+                    <p>
+                        You pay only for what you use. You are billed for the number of resources managed in your Pulumi
+                        account on a per-hour basis. Each partial hour used is billed as a full hour. If there are no
+                        resources in your account, then there is no usage for billing purposes.
+                    </p>
+                    <p>
+                        Hourly usage is measured in the number of Pulumi Credits, and you are billed 1 Pulumi Credit
+                        to manage 1 <a class="underline" href="{{ relref . "/docs/intro/concepts/resources" }}">resource</a>
+                        for 1 hour. Each credit costs $0.00025.
```
Comment (susanev) [inline themes/default/layouts/page/pricing.html:248]:
```suggestion
                        Hourly usage is measured in the number of Pulumi Credits, and you are billed one Pulumi Credit
                        to manage one <a class="underline" href="{{ relref . "/docs/intro/concepts/resources" }}">resource</a>
                        for one hour. Each credit costs $0.00025.
```

File: themes/default/layouts/page/pricing.html [hunk 14]
```diff
@@ -200,801 +180,305 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
...
+                    </p>
+                    <p>
+                        <span class="font-bold">350,000 Credits</span><br />
+                        estimated cost: $50/month<br />
+                        2 production environments, about 450 resources
+                    </p>
+                    <p>
+                        <span class="font-bold">1,000,000 Credits</span><br />
+                        estimated cost: $225/month<br />
+                        Production deployment with 5 environments, about 1,300 resources
```
Comment (susanev) [inline themes/default/layouts/page/pricing.html:300]:
```suggestion
                        one small production environment, about 200 resources
                    </p>
                    <p>
                        <span class="font-bold">350,000 Credits</span><br />
                        estimated cost: $50/month<br />
                        two production environments, about 450 resources
                    </p>
                    <p>
                        <span class="font-bold">1,000,000 Credits</span><br />
                        estimated cost: $225/month<br />
                        Production deployment with five environments, about 1,300 resources
```

File: themes/default/layouts/page/pricing.html [hunk 15]
```diff
@@ -200,801 +180,305 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
...
+                         at all for it.
+                    </p>
+                    <p>
+                        Second, Pulumi is an <a class="underline" href="https://github.com/pulumi/pulumi">open source project</a>. You can
+                        <a class="underline" href="{{ relref . "/docs/intro/concepts/state#self-managed-backend" }}">run Pulumi entirely
+                        offline</a> without the online service’s features, and manage state yourself, instead of using the online service.
+                        There are no restrictions &mdash; it's all there in the open for you to use freely as you'd like.
+                    </p>
+                    <p>
+                        Finally, Pulumi Team Edition offers a 30-day free trial. Once the trial is over, you get 150K free
```
Comment (susanev) [inline themes/default/layouts/page/pricing.html:340]:
```suggestion
                        Finally, Pulumi Team Edition offers a 30-day free trial. Once the trial is over, you get 150k free
```

File: themes/default/content/blog/announcing-new-usage-based-pricing-for-your-whole-team/index.md [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+
+Pricing based on the number of resources under management means that price scales naturally with the value of the cloud infrastructure complexity being tamed, managed and leveraged for your organization’s business goals via Pulumi.  Pricing is in terms of Pulumi Credits, and a Pulumi Credit is one [cloud resource]({{< relref "/docs/intro/concepts/resources" >}}) under management for one hour.
```
Comment (joeduffy) [inline themes/default/content/blog/announcing-new-usage-based-pricing-for-your-whole-team/index.md:20]:
```suggestion
Pricing based on the number of resources under management means that price scales naturally with the cloud infrastructure complexity being tamed, and aligns is well with how your infrastructure is purchased.  Pricing is based on Pulumi Credits, and a Pulumi Credit is one [cloud resource]({{< relref "/docs/intro/concepts/resources" >}}) under management for one hour.
```

File: themes/default/content/blog/announcing-new-usage-based-pricing-for-your-whole-team/index.md [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+* **Enterprise Edition**: for large organizations that need unlimited users, unlimited teams, Role-Based Access Control, SAML/SSO, policy enforcement, audit logging, and 12x5 or 24x7 support for their production deployments.
+
+Full details of the new editions are available on our [pricing page]({{< relref "/pricing" >}}).
+
+![Pricing Page](pricing-page.png)
+
+You can try out Pulumi with a 30-day free trial, no credit card required. If you need a longer trial period, want to discuss potential proof of concept projects, or are interested in advanced capabilities, such as policy and audit logs, Training, or Support, please [contact us]({{< relref "/contact" >}}).
+
+We remain committed to fostering a vibrant open source community while also delivering a robust production service that teams of all sizes are choosing for their cloud needs. From startups, to medium sized organizations, to the largest Fortune 500 customers, we truly believe that these new editions now offer a great option for everyone.
```
Comment (joeduffy) [inline themes/default/content/blog/announcing-new-usage-based-pricing-for-your-whole-team/index.md:36]:
```suggestion
We remain committed to fostering a vibrant open source community while also delivering a robust production service that teams of all sizes are choosing for their cloud needs. From startups, to medium sized organizations, to the largest Fortune 500 customers, we truly believe that these new editions now offer a great option for everyone. If you are using our pure open source offering, nothing has changed. We have also recently added a new offer for open source projects and foundations, and provide discounts for early stage startups. To discuss either, please contact us.
```

I don’t have the links handy for these — could someone add them?

File: themes/default/content/blog/announcing-new-usage-based-pricing-for-your-whole-team/index.md [hunk 18]
```diff
@@ -0,0 +1,38 @@
...
+
+We hope you agree and look forward to seeing the great things your team builds with Pulumi!
```
Comment (joeduffy) [inline themes/default/content/blog/announcing-new-usage-based-pricing-for-your-whole-team/index.md:38]:
```suggestion
We hope this change helps to get Pulumi into the hands of more of your team, and look forward to seeing the great things you and your team builds with Pulumi!
```

File: themes/default/content/blog/announcing-new-usage-based-pricing-for-your-whole-team/index.md (hunk 17 above)
Comment (susanev) [reply themes/default/content/blog/announcing-new-usage-based-pricing-for-your-whole-team/index.md:36]: i added the open source free tier, and contact us.. wasn't sure what else you wanted to link to..

**Pull request merged**